documents/emandate.md

```diff
@@ -418,7 +417,6 @@ instance.orders.create({
 | currency*        | string  | Currency of the order. Currently only `INR` is supported.                      |
 | receipt         | string  | Your system order reference id.                                              |
 | notes           | object  | A key-value pair                                                             |
-| payment_capture  | boolean  | Indicates whether payment status should be changed to captured automatically or not. Possible values: true - Payments are captured automatically. false - Payments are not captured automatically. |
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Shall we add details about partial_payments here now?

Reply:
partial_payments parameter is only required in order api doc also we need payment_capture for emandate and nach so i have added that back (confirmed with tech writer)